src/bin/lpython.cpp

```diff
@@ -1785,6 +1941,9 @@ int main(int argc, char *argv[])
         compiler_options.use_colors = !arg_no_color;
         compiler_options.indent = !arg_no_indent;
 
+        lpython_pass_manager.parse_pass_arg(arg_pass, skip_pass);
+        lpython_pass_manager.use_default_passes();
+
```

Comment:
Do you need this change for `interactivity`? 

It should not be here. `lpython_pass_manager.use_default_passes();` applies ASR passes to the output of `--show-asr`. This changes the reference results and therefore the failure at the CI.

Let's revert this back.

Reply:
`lpython_pass_manager.parse_pass_arg(arg_pass, skip_pass);` and `lpython_pass_manager.use_default_passes();` is required to be initilised before calling `interactive_python_repl`. I will remove it from here and put it just before calling `interactive_python_repl`.

Overall behavior will be the same as before.